package/MDAnalysis/topology/core.py

```diff
@@ -34,19 +34,5 @@
 
 """
 
-# Global imports
-import os.path
-import numpy as np
-from collections import defaultdict
-
-# Local imports
-from . import tables
-from .guessers import (
```

Comment:
I'm not sure if these are imported as part of the expected API, someone else would have to double check

Reply:
`topology.tables` seems to be expected to be present as part of the API https://docs.mdanalysis.org/stable/documentation_pages/topology/tables.html ; we can change it for 3.0 but for this PR, these imports need to stay

I am not 100% sure about guessers — usage in https://docs.mdanalysis.org/stable/documentation_pages/topology/guessers.html seems ambiguous or indicative that it's _not_ required. I also don't see examples in the [User Guide under Guessing](https://userguide.mdanalysis.org/stable/formats/guessing.html) using `mda.topology.guess_*()`  @lilyminium do you happen to know if we are promising anywhere that `mda.topology.guess_*()` exists? Could we remove these imports? I'd be in favor of removing them right now if they are not in the API.

Reply:
I don't *think* we promise this as an official recommendation of how to use it, although people may still be using the guesser functions and they are publically available. That being said, with #3753 these functions will go away anyway (as they'll come under the DefaultGuesser class) so I am in favour of removing them here.

Reply:
Thanks!

So then please do the following @vachram97 
- Keep `from . import tables`
- Remove `from .guessers import ( ... )`